boards/FSM/Src/App/App_Signals.c

```diff
@@ -0,0 +1,52 @@
+#include "App_AcceleratorPedal.h"
+#include "App_Signals.h"
+
+struct AcceleratorPedal *_papps;
+struct AcceleratorPedal *_sapps;
+
+static bool IsPappsAlarmActive(void)
+{
+    return App_AcceleratorPedal_IsEncoderAlarmActive(_papps);
+}
+
+static void PappsAlarmCallback(void *context)
+{
+    struct FsmWorld *         world  = (struct FsmWorld *)context;
+    struct FsmCanTxInterface *can_tx = App_FsmWorld_GetCanTx(world);
+
+    App_CanTx_SetPeriodicSignal_PAPPS_MAPPED_PEDAL_PERCENTAGE(can_tx, 0.0f);
+}
+
+static bool IsSappsAlarmActive(void)
+{
+    return App_AcceleratorPedal_IsEncoderAlarmActive(_papps);
```

Comment:
``` suggestion
    return App_AcceleratorPedal_IsEncoderAlarmActive(_sapps);
```
not sure how extensively we're testing this version of the shared library or the FSM specific stuff here, but might want to add at least one or two cases that would catch this

Reply:
Fixed!

Reply:
Turns out the problem lies in `App_AcceleratorPedal.c`

I wrote 
```
// This _always_ returns  true!
return accelerator_pedal->is_encoder_alarm_active;
```
instead of 
```
return return accelerator_pedal->is_encoder_alarm_active(); 
```

Kind of a silly bug. Turns out I just forgot to write `Test_AcceleratorPedal` so this wasn't caught :sweat_smile: